pkg/contract_watcher/types/method.go

```diff
@@ -94,7 +94,7 @@ func NewMethod(m abi.Method) Method {
 
 	return Method{
 		Name:   m.Name,
-		Const:  m.Const,
+		Const:  m.Constant,
```

Comment:
Is this necessary? If so, does it affect backward compatibility?

Reply:
good call. 

When we update the geth version that vdb relies on, then this will need to change because `Const` was changed to `Constants`. However, we shouldn't need to update the geth dependency this much at the moment. I'll change that shortly.